pkgs/development/compilers/llvm/default.nix

```diff
@@ -14,7 +14,12 @@
   bootBintoolsNoLibc ? if stdenv.targetPlatform.linker == "lld" then null else pkgs.bintoolsNoLibc,
   bootBintools ? if stdenv.targetPlatform.linker == "lld" then null else pkgs.bintools,
   llvmVersions ? { },
-}:
+  # Allows passthrough to packages via newScope in ./common/default.nix.
+  # This makes it possible to do
+  # `(llvmPackages.override { <someLlvmDependency> = bar; }).clang` and get
+  # an llvmPackages whose packages are overridden in an internally consistent way.
+  ...
+}@packageSetArgs:
```

Comment:
Just realising this should probably be called packageArgs rather than packageSet args, since llvm/default.nix is not itself a packageSet; will rename.

Reply:
Scratch that; I was confused and thought I was looking at LLVM itself.